interface/patient_file/history/encounters.php

```diff
@@ -297,6 +308,16 @@ function changePageSize() {
     </span>
 
     <br />
+    <span class="heading" >
+    <?php
+  // RM put patienes name, id and dob at top of the history -->
```

Comment:
This breaks if the attendant_type is not a patient (IE therapy group encounters are being viewed.  We need to surround all of this:
```
 if ($attendant_type == 'pid') {
```